lib/airbrake-ruby/notice.rb

```diff
@@ -64,6 +64,10 @@ class Notice
     #   in filters
     attr_reader :stash
 
+    ##
+    # @return [Exception] raw exception
+    attr_reader :exception
```

Comment:
I don't see why not. @kuboon, that will probably the preferred solution. Let me know what you think.